--- joe-4.6/rc/joerc.in.gnoterm	2018-01-10 17:28:34.000000000 +0100
+++ joe-4.6/rc/joerc.in	2018-01-10 23:16:55.177253705 +0100
@@ -73,7 +73,7 @@
 		after the path).
 
  -floatmouse	Clicking past end of line moves the cursor past the end
- 
+
  -rtbutton	Use the right mouse button for action, instead of the left
 
  -nonotice	Disable copyright notice
@@ -794,6 +794,8 @@ msg		^[ ^H		Display a message
 nextw		^K N		Goto next window
 nextw		^K ^N
 nextw		^K n
+nextw		^[ [ 1 ; 3 C	alt right in (newer) xterm
+nextw		^[ [ 3 C	alt right in gnome-terminal
 pgdn		.kN		Screen down
 pgdn		^V
 pgdn		^[ [ 6 ~
@@ -806,6 +808,8 @@ play		^K 0 TO 9	Execute macro
 prevw		^K P		Goto previous window
 prevw		^K ^P
 prevw		^K p
+prevw		^[ [ 1 ; 3 D    alt left in (newer) xterm
+prevw		^[ [ 3 D        alt left in gnome-terminal
 query		^K ?		Macro query insert
 record		^K [		Record a macro
  retype		^R		Refresh screen
@@ -844,8 +848,8 @@ extmouse	^[ [ <		Introduces an extended
 :def ispellword psh,nextword,markk,prevword,markb,filt,"cat >ispell.tmp;ispell ispell.tmp </dev/tty >/dev/tty;tr -d <ispell.tmp '\\012';/bin/rm ispell.tmp",rtn,retype,nextword
 
  Aspell
-:def aspellfile filt,"SPLTMP=ispell.tmp;cat >$SPLTMP;aspell --lang=",language,".",charset," -x -c $SPLTMP </dev/tty >/dev/tty;cat $SPLTMP;/bin/rm $SPLTMP",rtn,retype
-:def aspellword psh,nextword,markk,prevword,markb,filt,"SPLTMP=ispell.tmp;cat >$SPLTMP;aspell --lang=",language,".",charset," -x -c $SPLTMP </dev/tty >/dev/tty;tr -d <$SPLTMP '\\012';/bin/rm $SPLTMP",rtn,retype,nextword
+:def aspellfile filt,"SPLTMP=`mktemp -t joespell.XXXXXXXXXX`;cat >$SPLTMP;aspell --lang=",language,".",charset," -x -c $SPLTMP </dev/tty >/dev/tty;cat $SPLTMP;/bin/rm $SPLTMP",rtn,retype
+:def aspellword psh,nextword,markk,prevword,markb,filt,"SPLTMP=`mktemp -t joespell.XXXXXXXXXX`;cat >$SPLTMP;aspell --lang=",language,".",charset," -x -c $SPLTMP </dev/tty >/dev/tty;tr -d <$SPLTMP '\\012';/bin/rm $SPLTMP",rtn,retype,nextword
 
 
 aspellfile	^[ l
@@ -1133,9 +1137,13 @@ markk		^K k
 menu,"root",rtn	^T
 nextpos		^K =		Goto next position in position history
 nextword	^X		Goto next word
+nextword	^[ [ 1 ; 5 C	ctrl right in (newer) xterm
+nextword	^[ [ 5 C	ctrl right in gnome-terminal
 open		^]		Split line
 prevpos		^K -		Previous position in history
 prevword	^Z		Previous word
+prevword	^[ [ 1 ; 5 D	ctrl left in (newer) xterm
+prevword	^[ [ 5 D	ctrl left in gnome-terminal
 reload		^K `		Revert file
 redo		^^		Redo changes
 rindent		^K .		Indent to right
